Cargo.lock

```diff
@@ -797,16 +797,16 @@ dependencies = [
  "serde",
  "swc_atoms",
  "swc_common",
- "swc_ecmascript",
+ "swc_ecmascript 0.108.4",
```

Comment:
Hmmm

Reply:
I don't see why "swc_ecmascript" "0.108.4" is sticking around in this file. Maybe we need to run cargo update?

Reply:
Oh, deno_core needs updating. I wasn't searching correctly.